src/facade/dragonfly_connection.cc

```diff
@@ -732,7 +734,7 @@ auto Connection::IoLoop(util::FiberSocketBase* peer, SinkReplyBuilder* orig_buil
         DCHECK_GT(io_buf_.AppendLen(), 0U);
       } else if (io_buf_.AppendLen() == 0) {
         // We have a full buffer and we can not progress with parsing.
-        // This means that we have request too large.
+        // This means that we have a too large request.
```

Comment:
Actually sounds fine to me as `a request too large to be processed` is valid english 🙂